Data/Sys/GameSettings/GMSE01.ini

```diff
@@ -132,8 +95,86 @@ $D-Pad Left For Normal Mario
 4240E12C 4DE63F80
 4240E12C 4DE83F80
 00000000 40000000
-$Widescreen Heat Wave Fix [YoshiOG1]
-043AA050 3F2AAAAB
-043AA05C BF2AAAAB
-043AA060 3F2AAAAB
-043AA070 3FAAAAAB
+[Gecko]
+$Improved Jesus Mode DPad Up/Down = On/Off
```

Comment:
Yes, the code was called Jesus Mode originally, but it also included the clarification "A.K.A. Walk on Water". If you're going to continue using the name Jesus Mode, you should continue including that clarification.